app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -300,13 +315,43 @@ class SectionDateHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderBi
     }
 }
 
-class SectionConflictHeaderViewHolder(private val viewBinding: ItemTxSectionHeaderBinding) :
+class SectionConflictHeaderViewHolder(private val viewBinding: ItemTxConflictSectionHeaderBinding) :
     BaseTransactionViewHolder<TransactionView.SectionConflictHeader>(viewBinding) {
 
     override fun bind(sectionDateHeader: TransactionView.SectionConflictHeader, payloads: List<Any>) {
+        val resources = viewBinding.root.context.resources
+
         with(viewBinding) {
-            sectionTitle.text = sectionDateHeader.nonce.toString()
+            nonce.text = sectionDateHeader.nonce.toString()
+            sectionTitle.text = resources.getString(R.string.tx_list_conflict_header_explainer)
+            sectionTitle.appendLink(
+                resources.getString(R.string.tx_list_conflict_header_link),
+                resources.getString(R.string.tx_list_conflict_header_learn_more)
+            )
+        }
+    }
+
+    private fun TextView.appendLink(url: String, urlText: String) {
+        val textColor = ForegroundColorSpan(textColors.defaultColor) //TODO get foreground color from TextView
+
+        val linkDrawable = ContextCompat.getDrawable(context, R.drawable.ic_link_green_24dp)!!
+        linkDrawable.setBounds(0, 0, linkDrawable.intrinsicWidth, linkDrawable.intrinsicHeight)
+        movementMethod = LinkMovementMethod.getInstance()
+        text = SpannableStringBuilder()
+            .appendText(Html.fromHtml(text.toString()), textColor)
```

Comment:
I am not sure what happens here, are you appending the spannable to itself as a string? Why is this necessary?

Reply:
Thank you for pointing this out!  🚀  
No it's not necessary. In fact it is harmful. I changed it to use `TextView.append()` instead. And now several links can be appended without problems. And now I can make it public.